.github/workflows/build-qa.yml

```diff
@@ -39,8 +39,12 @@ jobs:
           IMAGE_TAG: ${{ github.sha }}
           AIRTABLE_KEY: ${{ secrets.AIRTABLE_API_KEY }}
           NEW_RELIC_LICENSE_KEY: ${{ secrets.NEW_RELIC_LICENSE_KEY }}
+          NEXT_PUBLIC_ADOBE_ANALYTICS: ${{ secrets.NEXT_PUBLIC_ADOBE_ANALYTICS }}
```

Comment:
this value doesn't need to be a secret, it's publicly available by looking at the DOM, so I would just inline it here